<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EquipmentWriter</class>
 <widget class="QWidget" name="EquipmentWriter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>340</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="_2">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="minimumSize">
      <size>
       <width>252</width>
       <height>0</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="Scope">
      <attribute name="title">
       <string>Telescope</string>
      </attribute>
      <layout class="QHBoxLayout" name="_3">
       <item>
        <widget class="QListWidget" name="ScopeList"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QFormLayout" name="formLayout">
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::ExpandingFieldsGrow</enum>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Id:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="Id">
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Vendor:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="Vendor"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Model:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="Model"/>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Type:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QComboBox" name="Type">
             <property name="focusPolicy">
              <enum>Qt::StrongFocus</enum>
             </property>
             <item>
              <property name="text">
               <string>Refractor</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Newtonian</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Maksutov</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Schmidt-Cassegrain</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Kutter (Schiefspiegler)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Cassegrain</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Aperture:</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QDoubleSpinBox" name="Aperture">
               <property name="focusPolicy">
                <enum>Qt::StrongFocus</enum>
               </property>
               <property name="maximum">
                <double>10000.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.100000000000000</double>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_15">
               <property name="text">
                <string>mm</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Focal Length:</string>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QDoubleSpinBox" name="FocalLength">
               <property name="focusPolicy">
                <enum>Qt::StrongFocus</enum>
               </property>
               <property name="maximum">
                <double>40000.000000000000000</double>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>mm</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="driverLabel">
             <property name="text">
              <string>Driver:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QComboBox" name="driverComboBox">
             <property name="focusPolicy">
              <enum>Qt::StrongFocus</enum>
             </property>
             <item>
              <property name="text">
               <string>None</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="AddScope">
             <property name="toolTip">
              <string>Save telescope information</string>
             </property>
             <property name="text">
              <string>Save</string>
             </property>
             <property name="icon">
              <iconset theme="list-add"/>
             </property>
             <property name="iconSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="NewScope">
             <property name="toolTip">
              <string>Clear data and add a new telescope</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset theme="edit-clear">
               <normaloff>.</normaloff>.</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="RemoveScope">
             <property name="toolTip">
              <string>Remove current telescope</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset theme="list-remove">
               <normaloff>.</normaloff>.</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Eyepiece">
      <attribute name="title">
       <string>Eyepiece</string>
      </attribute>
      <layout class="QHBoxLayout">
       <item>
        <widget class="QListWidget" name="EyepieceList"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QFormLayout" name="formLayout_2">
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::ExpandingFieldsGrow</enum>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Id:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="e_Id">
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Vendor:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="e_Vendor"/>
           </item>
           <item row="4" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QDoubleSpinBox" name="e_focalLength">
               <property name="focusPolicy">
                <enum>Qt::StrongFocus</enum>
               </property>
               <property name="singleStep">
                <double>0.100000000000000</double>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_17">
               <property name="text">
                <string>mm</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Focal Length:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="e_Model"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Model:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Apparent FOV:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="Fov">
             <property name="focusPolicy">
              <enum>Qt::StrongFocus</enum>
             </property>
             <property name="maximum">
              <double>180.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Unit:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="FovUnit">
             <item>
              <property name="text">
               <string>deg</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>rad</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>0</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="NewEyepiece">
             <property name="text">
              <string>Add New</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="AddEyepiece">
             <property name="text">
              <string>Save Eyepiece</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="RemoveEyepiece">
             <property name="text">
              <string>Remove</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Lens">
      <attribute name="title">
       <string>Lens</string>
      </attribute>
      <layout class="QHBoxLayout" name="_4">
       <item>
        <widget class="QListWidget" name="LensList"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout3">
         <item>
          <widget class="QLabel" name="label_16">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Enter details of Barlow/Shapley lenses</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QFormLayout" name="formLayout3">
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::ExpandingFieldsGrow</enum>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label3_2">
             <property name="text">
              <string>Id:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="l_Id">
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label31">
             <property name="text">
              <string>Vendor:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="l_Vendor"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label51">
             <property name="toolTip">
              <string>Magnification factor of the Barlow/Shapley lens</string>
             </property>
             <property name="text">
              <string>Factor:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QDoubleSpinBox" name="l_Factor">
             <property name="focusPolicy">
              <enum>Qt::StrongFocus</enum>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="l_Model"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label2">
             <property name="text">
              <string>Model:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout3">
           <item>
            <spacer name="horizontalSpacer31">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="NewLens">
             <property name="text">
              <string>Add New...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="AddLens">
             <property name="text">
              <string>Save Lens</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="RemoveLens">
             <property name="text">
              <string>Remove</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Filter">
      <attribute name="title">
       <string>Filter</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QListWidget" name="FilterList"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QFormLayout" name="formLayout_3">
           <item row="0" column="0">
            <widget class="QLabel" name="label4">
             <property name="text">
              <string>Id:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="f_Id">
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label5">
             <property name="text">
              <string>Vendor:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="f_Vendor"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label3">
             <property name="text">
              <string>Model:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="f_Model"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label6">
             <property name="text">
              <string>Type:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="f_Type"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Color:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="f_Color"/>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_19">
             <property name="text">
              <string>Exposure:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QDoubleSpinBox" name="f_Exposure">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimum">
              <double>0.010000000000000</double>
             </property>
             <property name="maximum">
              <double>120.000000000000000</double>
             </property>
             <property name="value">
              <double>1.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_18">
             <property name="toolTip">
              <string>Filter focus offset</string>
             </property>
             <property name="text">
              <string>Offset:</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QSpinBox" name="f_Offset">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimum">
              <number>-10000</number>
             </property>
             <property name="maximum">
              <number>10000</number>
             </property>
             <property name="singleStep">
              <number>100</number>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="label_21">
             <property name="text">
              <string>Auto Focus:</string>
             </property>
            </widget>
           </item>
           <item row="8" column="1">
            <widget class="QCheckBox" name="f_UseAutoFocus">
             <property name="text">
              <string>Enable</string>
             </property>
            </widget>
           </item>
           <item row="9" column="0">
            <widget class="QLabel" name="label_20">
             <property name="text">
              <string>Locked Filter:</string>
             </property>
            </widget>
           </item>
           <item row="9" column="1">
            <widget class="QLineEdit" name="f_LockedFilter"/>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="label_22">
             <property name="text">
              <string>Abs. Position:</string>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="QSpinBox" name="f_AbsoluteFocusPosition">
             <property name="maximum">
              <number>1000000</number>
             </property>
             <property name="singleStep">
              <number>1000</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout7">
           <item>
            <spacer name="horizontalSpacer3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="NewFilter">
             <property name="text">
              <string>Add New...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="AddFilter">
             <property name="text">
              <string>Save Filter</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="RemoveFilter">
             <property name="text">
              <string>Remove</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>Vendor</tabstop>
  <tabstop>Model</tabstop>
  <tabstop>driverComboBox</tabstop>
  <tabstop>Type</tabstop>
  <tabstop>Aperture</tabstop>
  <tabstop>FocalLength</tabstop>
  <tabstop>AddScope</tabstop>
  <tabstop>RemoveScope</tabstop>
  <tabstop>EyepieceList</tabstop>
  <tabstop>e_Vendor</tabstop>
  <tabstop>e_Model</tabstop>
  <tabstop>e_focalLength</tabstop>
  <tabstop>Fov</tabstop>
  <tabstop>FovUnit</tabstop>
  <tabstop>NewEyepiece</tabstop>
  <tabstop>AddEyepiece</tabstop>
  <tabstop>RemoveEyepiece</tabstop>
  <tabstop>LensList</tabstop>
  <tabstop>l_Vendor</tabstop>
  <tabstop>l_Model</tabstop>
  <tabstop>l_Factor</tabstop>
  <tabstop>NewLens</tabstop>
  <tabstop>AddLens</tabstop>
  <tabstop>RemoveLens</tabstop>
  <tabstop>FilterList</tabstop>
  <tabstop>f_Vendor</tabstop>
  <tabstop>f_Model</tabstop>
  <tabstop>f_Type</tabstop>
  <tabstop>NewFilter</tabstop>
  <tabstop>AddFilter</tabstop>
  <tabstop>ScopeList</tabstop>
  <tabstop>RemoveFilter</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
